tests/synoptic_tests.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from tests.settings import Settings
+from util.gui import GuiUtils
+from util.synoptic import SynopticUtils
+from util.version import VersionUtils
+
+
+class SynopticTests(unittest.TestCase):
+
+    def __init__(self, methodName, synoptic=None):
```

Comment:
Default option as per components and configurations. You don't `assert is not None` here, unlike elsewhere. Is that intentional?